corehq/apps/products/fixtures.py

```diff
@@ -52,8 +53,19 @@ def get_products():
                 key=lambda product: product.code
             )
 
-        return simple_fixture_generator(
+        fixture_nodes = simple_fixture_generator(
             restore_user, self.id, "product", PRODUCT_FIELDS, get_products, restore_state.last_sync_log
         )
+        if not fixture_nodes:
+            return []
+
+        if restore_state.params.openrosa_version and restore_state.params.openrosa_version < '2.1':
+            # Don't include index schema when openrosa version is specified and below 2.1
```

Comment:
Just added it to `OPENROSA_VERSION_MAP`.

I think the thought is that the "correct" way to do this is to always include the schema, and older versions will just ignore it.  The first couple versions using the schema have a bug which means we can't provide this optimization, but those are problem versions, the versions older and new versions are fine.  With that framing, I think this logic makes sense, and for that reason I also kept the "bad" `if` branch first, as a special case that shortcuts out early.